src/System.Net.HttpListener/tests/HttpListenerRequestTests.cs

```diff
@@ -171,8 +180,7 @@ public static IEnumerable<object[]> ContentEncoding_TestData()
             });
         }
         
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
-        [PlatformSpecific(TestPlatforms.Windows)]
+        [ConditionalFact(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))]
```

Comment:
Is there an issue number for this?  Let's try to ensure that anywhere we're special-casing out a platform, we have a comment that indicates why, either an issue number if it's temporary or a short-description of why it's permanently platform-specific.